test/functional/operation_promises_example_tests.js

```diff
@@ -2822,10 +2822,10 @@ exports.shouldCorrectlyLogoutFromTheDatabaseWithPromises = {
 
               db.close();
               test.done();
-            }).catch(function(err) {  })
-          }).catch(function(err) {  })
-        }).catch(function(err) {  })
-      }).catch(function(err) {  })
+            }).catch(function(err) {  console.dir(err )})
```

Comment:
I think we tackle this in the bigger scope of "rewrite" tests at some point later. There are a lot dodgy tests like that.